test/e2e/network/netpol/model.go

```diff
@@ -181,6 +187,20 @@ type Pod struct {
 	Namespace  string
 	Name       string
 	Containers []*Container
+	ServiceIP string
+}
+
+func (m *Model) SetServiceIP(namespace string, name string, ip string) {
```

Comment:
Shouldn't this method be on `(p *Pod)` instead of `Model`?  Every caller has a pod already, this method just adds error conditions?

Reply:
done